docs/design.md

```diff
@@ -157,19 +266,26 @@ Blindly return:
   rpc:
```

Comment:
Can we change this to gRPC

Reply:
I'm pretty sure this is what the driver actually returns.

See [CSI Spec ControllerGetCapabilities response struct](https://github.com/container-storage-interface/spec/blob/master/spec.md#controllergetcapabilities)

Reply:
gotcha will close accordingly